salvo-remote/sandboxes/sandboxes_test.go

```diff
@@ -113,33 +134,206 @@ func TestStart(t *testing.T) {
 			applyErr:      errors.New("fake error"),
 			wantErrSubstr: "tf.Apply =>",
 		},
+		{
+			desc: "fails when terraform output fails",
+			sbxs: map[Type]Instances{
+				TypeDefaultSandboxX64: {12345},
+			},
+			outputErr:     errors.New("fake error"),
+			wantErrSubstr: "tf.Output =>",
+		},
 		{
 			desc: "successfully starts a sandbox instance",
 			sbxs: map[Type]Instances{
 				TypeDefaultSandboxX64: {12345},
 			},
+			output: map[string]tfexec.OutputMeta{
+				defSbxX64Gen0ControlIPs: {
+					Value: json.RawMessage(`
+					{
+						"12345": ["1.1.1.1"]
+					}
+`),
```

Comment:
Can these "`)," be indented?

Reply:
All Go files are actually formatted automatically, so we cannot (and should not) tweak the formatting. This is how the gofmt formats it.

Go made a decision to leave all formatting to the machine - https://go.dev/doc/effective_go#formatting

Reply:
@mum4k Got it, thank you for your clarification.